pipeline/matcher/src/main/scala/uk/ac/wellcome/platform/matcher/Main.scala

```diff
@@ -41,30 +45,21 @@ object Main extends WellcomeTypesafeApp {
       )
     )
 
-    val rowLockDaoConfig = DynamoRowLockDaoConfig(
-      dynamoConfig =
+    implicit val lockDao = new DynamoLockDao(
+      dynamoClient,
+      DynamoLockDaoConfig(
         DynamoBuilder.buildDynamoConfig(config, namespace = "locking.service"),
-      duration = Duration.ofSeconds(180)
-    )
-
-    val lockingService = new DynamoLockingService(
-      lockNamePrefix = "WorkMatcher",
-      dynamoRowLockDao = new DynamoRowLockDao(
-        dynamoDbClient = dynamoClient,
-        rowLockDaoConfig = rowLockDaoConfig
-      ),
-      metricsSender = MetricsBuilder.buildMetricsSender(config)
+        Duration.ofSeconds(180)
+      )
     )
 
-    val workMatcher = new WorkMatcher(
-      workGraphStore = workGraphStore,
-      lockingService = lockingService
-    )
+    val workMatcher = new WorkMatcher(workGraphStore, new DynamoLockingService)
 
     new MatcherWorkerService(
-      messageStream =
-        MessagingBuilder.buildMessageStream[TransformedBaseWork](config),
-      snsWriter = SNSBuilder.buildSNSWriter(config),
+      msgStream = BigMessagingBuilder
+        .buildMessageStream[TransformedBaseWork](config),
+      msgSender = SNSBuilder
+        .buildSNSMessageSender(config, subject = "Sent from the matcher"),
```

Comment:
Complete bikeshed

I wonder once we get logging into the pipeline steps homogeneously if we could [tag these messages](https://docs.aws.amazon.com/sns/latest/dg/sns-message-attributes.html).

That ay our logging could easily use those and send them through to logstash / ES?